ui/src/app/cohort-search/selection-list/selection-list.component.tsx

```diff
@@ -144,9 +145,10 @@ interface Props {
   selections: Array<Selection>;
   setView: Function;
   view: string;
+  criteria?: Array<Selection>;
 }
 
-export class SelectionList extends React.Component<Props> {
+export const SelectionList = withCurrentCohortCriteria()(class extends React.Component<Props> {
```

Comment:
The changes in this component will break the non feature flagged version of cohort builder. We either need to put the changes behind the feature flag or duplicate the component so we can change it however we want. Duplicating is probably simpler.

Reply:
yeah thats a good point , i will update this